src/mon/Elector.cc

```diff
@@ -117,8 +117,14 @@ void Elector::defer(int who)
   ack_stamp = ceph_clock_now();
   MMonElection *m = new MMonElection(MMonElection::OP_ACK, epoch, mon->monmap);
   m->mon_features = ceph::features::mon::get_supported();
-  m->sharing_bl = mon->get_local_commands_bl();
   mon->collect_metadata(&m->metadata);
+
+  // This field is unused completely in luminous, but jewel uses it to
+  // determine whether we are a dumpling mon due to some crufty old
+  // code.  It only needs to see this buffer non-empty, so put
+  // something useless there.
+  m->sharing_bl.append("I am not dumpling");
```

Comment:
I originally removed it because the command list is a function of which mon features the quorum has, and we don't know that here.  But then I added this back because then jewel thought we were dumpling mons.